src/include/rados/librados.hpp

```diff
@@ -1154,6 +1154,9 @@ namespace librados
 
     config_t cct();
 
+    void set_honor_osdmap_full();
+    void unset_honor_osdmap_full();
+
```

Comment:
Hmm, in that case, the FULL_TRY flag can be set in the same place as the FULL_FORCE flag: Objecter::_prepare_osd_op().  So you can add {set,unset}_osdmap_full_try(), and set the flag in one place, and then use that.

I really think we should use FULL_TRY unless we find there is some reason that can't work... if that's the case we have a bigger problem!